### ISSUE TITLE
NSwag and IdentityServer version conflict in .Net 6

### ISSUE THREAD

**Issue by bsurprised** [RicoSuter/NSwag#3702]
If the .Net 6 project includes the identity server, there's a conflict for the package `Microsoft.Extensions.FileProviders.Embedded`

```
Error	NU1107	Version conflict detected for Microsoft.Extensions.FileProviders.Embedded.
... Microsoft.AspNetCore.ApiAuthorization.IdentityServer 6.0.0 -> Microsoft.AspNetCore.Identity.UI 6.0.0 -> Microsoft.Extensions.FileProviders.Embedded (>= 6.0.0) 

 ... -> NSwag.AspNetCore 13.13.2 -> Microsoft.Extensions.FileProviders.Embedded (>= 5.0.0 && < 6.0.0).
```

**Comment by bsurprised** [comment]
I guess #3701 should fix this.

**Comment by maxkoshevoi** [comment]
Duplicate of #3699

**Comment by RicoSuter** [comment]
NSwag with .NET 6 support is released: Try v13.14.0

**Comment by jpflick** [comment]
Cannot run NSwagStudio or command line with 13.14.0. No Net60 binaries folder in C:\Program Files (x86)\Rico Suter\NSwagStudio or C:\Users\john-dev\AppData\Roaming\npm\node_modules\nswag\bin\binaries. Not sure if that is the problem. 

Error: Command failed: dotnet "C:\Users\john-dev\AppData\Roaming\npm\node_modules\nswag\bin/binaries/Net60/dotnet-nswag.dll" run nswag.json /runtime:Net60
